evol/population.py

```diff
@@ -7,7 +7,7 @@
 from itertools import cycle, islice
 from typing import Any, Callable, Generator, Iterable, Iterator, Optional, Sequence
 from uuid import uuid4
-
+from multiprocess import Pool
```

Comment:
Please use `from multiprocess.pool import Pool` instead. This is more IDE-friendly due to the dynamic assignment of `__all__` in `multiprocess/__init__.py`.